### PR TITLE
Added filled wardrobes for most jobs

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Right now all wardrobes are empty, so it can be very hard for people to join new departments without looking super sussy. So I added filled wardrobes. These still need to be mapped in, but they were sorely needed.

### CODE REVIEW

File: Resources/Prototypes/Catalog/Fills/Lockers/wardrobe_job.yml [hunk 1]
```diff
@@ -29,47 +53,173 @@
...
+       - id: ClothingShoesColorBrown
+         amount: 1
+       - id: ClothingOuterApronBotanist
+         amount: 1
+       - id: ClothingUniformJumpskirtHydroponics
+         amount: 1
+       - id: ClothingUniformOveralls
+         amount: 1
+       - id: ClothingHeadHatTrucker #the hat itself is a subtle joke
+         prob: 0.3
```
Comment (mirrorcult) [inline Resources/Prototypes/Catalog/Fills/Lockers/wardrobe_job.yml:101]: basado. maybe less than 0.3 chance though

File: Resources/Prototypes/Catalog/Fills/Lockers/wardrobe_job.yml [hunk 2]
```diff
@@ -29,47 +53,173 @@
...
-#   components:
-#   - type: StorageFill
-#     contents:
-#       - id: ClothingHandsGlovesLatex
-#         prob: 0.4
+ - type: entity
+   id: WardrobeChapelFilled
+   suffix: Filled
+   parent: WardrobeChapel
+   description: "Modest is hottest."
```
Comment (mirrorcult) [inline Resources/Prototypes/Catalog/Fills/Lockers/wardrobe_job.yml:132]: idk about this one

File: Resources/Prototypes/Catalog/Fills/Lockers/wardrobe_job.yml [hunk 3]
```diff
@@ -29,47 +53,173 @@
...
+       - id: ClothingOuterHoodieChaplain
+         amount: 1
+       - id: ClothingHeadHatFez
+         prob: 0.5
+       - id: ClothingHeadHatPlaguedoctor
+         prob: 0.15
+       - id: ClothingHeadHatWitch
+         prob: 0.05
+       - id: ClothingHeadHatWitch1
+         prob: 0.05
```
Comment (mirrorcult) [inline Resources/Prototypes/Catalog/Fills/Lockers/wardrobe_job.yml:155]: these should probably be in the chaplain vendor or w/e right?

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Lockers/wardrobe_job.yml (hunk 3 above)
Comment (Elijahrane) [reply Resources/Prototypes/Catalog/Fills/Lockers/wardrobe_job.yml:155]: Do we have that? If it should probably be mapped.

**Comment by Elijahrane** [comment]
I restored the piety vendor too with /tg/ sprites, shame I can't get my custom deny animation to work for some reason.

**Review by mirrorcult — APPROVED** [review]

**Review by PaulRitter — APPROVED** [review]

**Pull request merged**